bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/handler/DoorbellHandler.java

```diff
@@ -108,35 +98,33 @@ public DoorbirdHandler(Thing thing, TimeZoneProvider timeZoneProvider, HttpClien
 
     @Override
     public void initialize() {
-        updateStatus(ThingStatus.UNKNOWN);
-
-        config = getConfigAs(DoorbirdConfiguration.class);
-        if (StringUtils.isEmpty(config.doorbirdId)) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Doorbird id not provided");
-            return;
-        }
-        if (StringUtils.isEmpty(config.doorbirdHost)) {
+        String host = getConfigAs(DoorbellConfiguration.class).doorbirdHost;
```

Comment:
Same questions as above: Why don't you get the config once?

Reply:
Done